lib/rubocop/cop/naming/ascii_identifiers.rb

```diff
@@ -3,7 +3,39 @@
 module RuboCop
   module Cop
     module Naming
+      # rubocop:disable Style/AsciiComments
```

Comment:
Move this to the beginning of the class and remove the enable, so they won't appear in the manual. Probably we should fix the generator to simply exclude those lines, as we've had this problem a few times recently.